src/mftscleanup/business_days.py

```diff
@@ -19,11 +30,41 @@ def add_business_days(start: date, num_businessdays: int) -> date:
         current_date += ONE_DAY
         if current_date.weekday() >= 5:  # sunday = 6
             continue  # weekend, so nothing to do
-        if current_date in us_holidays:
-            continue  # businessday (wip)
-        if current_date.month == 11:  # November
-            if current_date.weekday() == 4:  # Friday
-                if 23 <= current_date.day <= 29:  # In range of Black Friday
-                    continue
+        if current_date in dep_holidays: 
+           continue # working day
         remaining_business_days -= 1
     return current_date
+
+
+class departmental_Holidays(holidays.US):
```

Comment:
Classes should use camel case, not snake case.

https://peps.python.org/pep-0008/#class-names

Reply:
Thanks, I didn't know what snake case was until now.

Reply:
f3dac209c494966fc9e61c23ac7e8a6a3aa3dadf

Reply:
That means capitalize the "D" at the start too.

Reply:
370ca04be707c63daad9cc8ab9d45412b079268e